src/pages/on-chain-governance/_components/DisclosureComponent.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { PropsWithChildren } from "react";
+import { MdEdit, MdCheckCircle } from "react-icons/md";
+import { Disclosure, Transition } from "@headlessui/react";
+import classNames from "classnames";
+
+export function DisclosureComponent({
+  title,
+  children,
+  isOpen,
+  isCompleted = false,
+  onEdit,
+}: PropsWithChildren<{
+  title: string;
+  isOpen: boolean;
+  isCompleted?: boolean;
+  onEdit?: () => void;
+}>) {
+  return (
+    <div className="py-5 px-6 md:px-10 rounded-lg border border-gray-200 dark:border-gray-700">
+      <Disclosure defaultOpen>
+        <div className={classNames({ "py-1 md:py-5": isOpen })}>
```

Comment:
```suggestion
        <div className={classNames({ "py-5": isOpen })}>
```

Reply:
Thanks for pointing out, Design is inconsistent for this